Automattic/vaultpress
# Fix Travis configuration

Travis should now send Slack notifications every time a build fails

**To test**

1. See documentation here:
https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications
2. Ensure format is the same (not like I did at first, I had copied/pasted what was given to me on Slack's end when creating the integration 🤦‍♂ )
